multiversx_sdk/abi/abi.py

```diff
@@ -277,6 +277,8 @@ def _create_prototype(self, type_formula: TypeFormula) -> Any:
             return AddressValue()
         if name == "TokenIdentifier":
             return TokenIdentifierValue()
+        if name == "EgldOrEsdtTokenIdentifier":
```

Comment:
With respect to:

https://github.com/multiversx/mx-sdk-js-core/blob/v13.4.0/src/smartcontracts/typesystem/typeMapper.ts